k8s-node-termination-handler/handler.libsonnet

```diff
@@ -1,105 +1,9 @@
-        [
-          nodeSelector.nodeSelectorTermsType.new() +
-          nodeSelector.nodeSelectorTermsType.withMatchExpressions([
-            nodeSelector.nodeSelectorTermsType.matchFieldsType.withKey('cloud.google.com/gke-accelerator')
-            + nodeSelector.nodeSelectorTermsType.matchFieldsType.withOperator('Exists'),
-          ]),
-          nodeSelector.nodeSelectorTermsType.new() +
-          nodeSelector.nodeSelectorTermsType.withMatchExpressions([
-            nodeSelector.nodeSelectorTermsType.matchFieldsType.withKey('cloud.google.com/gke-preemptible')
-            + nodeSelector.nodeSelectorTermsType.matchFieldsType.withOperator('Exists'),
-          ]),
-        ]
-      ) +
-      daemonSet.mixin.metadata.withNamespace(_config.namespace) +
-      daemonSet.mixin.spec.template.spec.withHostPid(true) +
-      daemonSet.mixin.spec.template.spec.withTolerations([
-        tolerations.new() +
-        tolerations.withOperator('Exists') +
-        tolerations.withEffect('NoSchedule'),
-
-        tolerations.new() +
-        tolerations.withOperator('Exists') +
-        tolerations.withEffect('NoExecute'),
-      ]),
-
-    local serviceAccount = k.core.v1.serviceAccount,
-    service_account:
-      serviceAccount.new('node-termination-handler') +
-      serviceAccount.mixin.metadata.withNamespace(_config.namespace),
-
-    local clusterRole = k.rbac.v1.clusterRole,
-    local clusterRoleRule = k.rbac.v1.clusterRole.rulesType,
-    cluster_role:
-      clusterRole.new() +
-      clusterRole.mixin.metadata.withName('node-termination-handler-role') +
-      clusterRole.withRulesMixin([
-        clusterRoleRule.new() +
-        clusterRoleRule.withApiGroups('') +
-        clusterRoleRule.withResources(['nodes']) +
-        clusterRoleRule.withVerbs(['get', 'update']),
-        clusterRoleRule.new() +
-        clusterRoleRule.withApiGroups('') +
-        clusterRoleRule.withResources(['events']) +
-        clusterRoleRule.withVerbs(['create']),
-        clusterRoleRule.new() +
-        clusterRoleRule.withApiGroups('') +
-        clusterRoleRule.withResources(['pods']) +
-        clusterRoleRule.withVerbs(['get', 'list', 'delete']),
-      ]),
-
-    local clusterRoleBinding = k.rbac.v1.clusterRoleBinding,
-    cluster_role_binding:
-      clusterRoleBinding.new() +
-      clusterRoleBinding.mixin.metadata.withName('node-termination-handler-role-binding') +
-      clusterRoleBinding.mixin.roleRef.withApiGroup('rbac.authorization.k8s.io') +
-      clusterRoleBinding.mixin.roleRef.withKind('ClusterRole') +
-      clusterRoleBinding.mixin.roleRef.withName('node-termination-handler-role') +
-      clusterRoleBinding.withSubjectsMixin({
-        kind: 'ServiceAccount',
-        name: 'node-termination-handler',
-        namespace: _config.namespace,
-      }),
-  },
+    } + config;
+    (import 'gke.libsonnet'),
```

Comment:
is this line for back-compat?

Reply:
yes: https://github.com/grafana/jsonnet-libs/pull/286/commits/11997d1e0623bf09cd83d0679ee24e901f1285a6

Reply:
mistake made, new commit added